File: TugasPraktikum06/Bahasa Python/PRAK603-2310817320013-AikoAnatashaWendiono.py
angka1, angka2 = map(int, input().split())

if angka1 != angka2:
    print("Jumlah tidak sama")
else:
    baris1 = list(map(int, input().split()))
    baris2 = list(map(int, input().split()))

    total = [baris1[i] * baris2[i] for i in range(angka1)]
    print(" ".join(map(str, total)))

File: TugasPraktikum06/Bahasa C/PRAK603-2310817320013-AikoAnatashaWendiono.c
#include <stdio.h>

int main() {
    int angka1, angka2;

    scanf("%d %d", &angka1, &angka2);

    if (angka1 != angka2) {
        printf("\nJumlah tidak sama\n");
    } else {
        int hasil[2 * angka1];

        for (int i = 0; i < 2 * angka1; ++i) {
            scanf("%d", &hasil[i]);
        } 
    printf("\n");

        for (int i = 0; i < angka1; ++i) {
            printf("%d ", hasil[i] * hasil[i + angka1]);
        } 
    }
    return 0;
}
